python/cudf/cudf/io/parquet.py

```diff
@@ -164,8 +164,7 @@ def read_parquet(
     filepath_or_buffer,
     engine="cudf",
     columns=None,
-    row_group=None,
-    row_group_count=None,
+    row_group_list=None,
```

Comment:
I'm a big fan of this change :)

Reply:
Can we rename this to `row_groups` instead? Presumably we can still pass a scalar integer here?

Reply:
done